lib/gitarro/backend.rb

```diff
@@ -7,6 +7,35 @@
 require_relative 'opt_parser'
 require_relative 'git_op'
 
+# this module have helper methods for changelog tests
+# it will be removed soon, but it helps to extract all
+# changelog code from backend class
+module ChangelogTests
```

Comment:
Nicely done. However this is not only fixing #67 but it is also work for #66 :-)

Reply:
yes, it is a small change. :)